docs/learning/geometry/BSpline.md

```diff
@@ -0,0 +1,198 @@
+# BSpline Curves
+
+A `BSplineCurve3d` (or `BSplineCurve3dH`) is a curve that (loosely) follows a sequence of control points (poles).
+
+Internally, the curve is a sequence of polynomial curves that join together smoothly. Call each of those separate pieces a _span_.
+
+A BSpline curve of order `k+1` (degree `k`) is defined by:
+$$
+S(t) = \sum_{i=0}^{n} N_{i,k}(t) P_i
+$$
```

Comment:
FYI, when I [preview](https://github.com/iTwin/itwinjs-core/blob/c8a78bd667fb59422d1f2d838e44ae248543cd52/docs/learning/geometry/BSpline.md) this file, GitHub isn't turning this into equations. I don't know why that is, since [this](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions) page claims that it should work.

Reply:
I changed `$$` to ` ```math ` and now GitHub renders latex as expected

https://github.com/iTwin/itwinjs-core/blob/de3dffbcd938b339c55a15ff09b24b3ed922e4ed/docs/learning/geometry/BSpline.md